CMakeLists.txt

```
@@ -13,8 +13,10 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 
-# TODO: re-enable /GL and /LTCG when we get a bigger build machine
-add_compile_options(/await /bigobj /GR- /permissive-)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

I think we can simplify this to the following:

```suggestion
if (MSVC)
```